lib/pbench/server/indexing_tarballs.py

```diff
@@ -425,13 +439,33 @@ def sigquit_handler(*args):
 
                         if sigquit_interrupt[0]:
                             break
+                        if sighup_interrupt[0]:
+                            status, new_tb = self.collect_tb()
+                            if status == 0 and new_tb:
+                                if not set(new_tb).issuperset(tb_deque):
+                                    idxctx.logger.info(
+                                        "Unexpected Situation: ",
+                                        set(tb_deque).difference(new_tb),
+                                    )
```

Comment:
I don't expect this to happen, and I'm not sure we need another iteration of the PR to fix it, but a little more context might have been useful if this message ever _does_ appear. Some of the tarballs we thought we still had to process are no longer showing as `TO-INDEX`, and that's puzzling. Simply listing the "missing" tarballs makes this message a bit more mysterious than it needs to be.

Reply:
I was quite not sure about the log message I should put. So. I will just change that